tardis/model/base.py

```diff
@@ -145,6 +269,40 @@ def elemental_number_density(self):
         )
 
 
+class ModelState_Experimental:
```

Comment:
If it is implemented into the standard Radial1DModel, perhaps this is no longer experimental? @wkerzendorf